codeHF/config_input.sh

```diff
@@ -78,7 +78,7 @@ case $INPUT_CASE in
     ISINPUTO2=1
     ISMC=1;;
   12)
-    INPUT_LABEL="p-p MC 13 TeV Xic, LHC19g6f"
+    INPUT_LABEL="p-p MC LHC19g6f" # dedicated Xic run2 MC (pp @ 13 TeV)"
```

Comment:
You should put the data period here, not the MC production one.

Reply:
Done, sorry for the misunderstanding.

Reply:
Thanks